src/Microsoft.AspNetCore.Blazor.Browser/Rendering/BrowserRendererRegistry.cs

```diff
@@ -12,18 +12,26 @@ namespace Microsoft.AspNetCore.Blazor.Browser.Rendering
     /// their associated component instances aren't GCed when events may still
     /// be received for them.
     /// </summary>
-    internal static class BrowserRendererRegistry
+    internal class BrowserRendererRegistry
     {
-        private static int _nextId;
-        private static IDictionary<int, BrowserRenderer> _browserRenderers
+        private int _nextId;
+        private IDictionary<int, BrowserRenderer> _browserRenderers
```

Comment:
We will likely need to make this thread-safe as well in the future.

Reply:
AFAICT it already is, but please let me know if I'm missing something!

Reply:
What I mean is that using a `BrowserRendererRegistry` is not thread safe because `Dictionary<,>` is not thread safe.

Reply:
The dictionary is only read/written from inside `lock` blocks. That should be sufficient, isn't it?

Reply:
Oh you're right, I'm not sure how I missed that sorry